public/components/FeatureAnywhereContextMenu/AnywhereParentFlyout/AnywhereParentFlyout.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import './styles.scss';
+import { get } from 'lodash';
+import AddAnomalyDetector from '../CreateAnomalyDetector';
+
+const AnywhereParentFlyout = ({ startingFlyout, ...props }) => {
+  const { embeddable } = props;
+  const indices: { label: string }[] = [
+    { label: get(embeddable, 'vis.data.indexPattern.title', '') },
+  ];
+  const [mode, setMode] = useState(startingFlyout);
+  const [selectedDetectorId, setSelectedDetectorId] = useState();
+
+  const Flyout = {
+    create: AddAnomalyDetector,
+  }[mode];
```

Comment:
Can we rename this to something more specific here? `Flyout` seems too generic and could get confused with OUI components